docs/usage.rst

```diff
@@ -161,6 +161,47 @@ functions all allow passing additional data to be tagged onto the error.
         Raven.setExtraContext({ foo: "bar" })
 
 
+.. _raven-js-recording-breadcrumbs:
+
+Recording Breadcrumbs
+---------------------
+
+Breadcrumbs are browser and application lifecycle events that are helpful in understanding the state of the application
+leading up to a crash.
+
+By default, Raven.js instruments browser built-ins and DOM events to automatically collect a few useful breadcrumbs
+for you:
+
+  * XMLHttpRequests
```

Comment:
We list 5 thing here but 4 options on the [config page](https://docs.sentry.io/clients/javascript/config/), missing a config option corresponding to previous errors. Is this configurable? Should it be? Do we report prior errors as breadcrumbs when autobreadcrumbs are otherwise disabled?